internal/resource/configmap_test.go

```diff
@@ -362,3 +375,13 @@ ssl_options.verify                       = verify_peer`)))
 		})
 	})
 })
+
+// iniString formats the input string using "gopkg.in/ini.v1"
+func iniString(input string) string {
+	ini.PrettySection = false
+	var output bytes.Buffer
+	cfg, _ := ini.Load([]byte(input))
+	_, err := cfg.WriteTo(&output)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
You can use [`ExpectWithOffset(1,...)` ](https://godoc.org/github.com/onsi/gomega#ExpectWithOffset) here so that the stack trace can point to where this helper is called :)